whylog/config/abstract_config.py

```diff
@@ -156,21 +155,30 @@ def _create_steps_in_investigation(self, concatenated_parsers, suspected_rules,
         return steps
 
     def _get_search_ranges(self, suspected_rules, effect_clues):
-        #TODO: remove mock
-        return {
-            'database': {
-                'date': {
-                    'left_bound': datetime(2016, 4, 12, 23, 53, 3),
-                    'right_bound': datetime(2016, 4, 12, 23, 54, 33)
-                }
-            },
-            'apache': {
-                'date': {
-                    'left_bound': datetime(2016, 4, 12, 23, 54, 33),
-                    'right_bound': datetime(2016, 4, 12, 23, 54, 43)
-                }
-            }
-        }
+        # This implementation assumes that all primary key groups is a one element list
```

Comment:
Please add docstring better describing this function that this comments about additional assumptions.